keadm/cmd/keadm/app/cmd/util/image.go

```diff
@@ -283,6 +286,9 @@ func (runtime *CRIRuntime) CopyResources(edgeImage string, files map[string]stri
 			},
 		},
 	}
+	if runtime.cgroupDriver == v1alpha2.CGroupDriverSystemd {
+		psc.Linux.CgroupParent = "/pods.slice/pods-copyresource.slice/pods-copyresource-pod.slice"
```

Comment:
Seems `/pods.slice/pods-copyresource.slice/pods-copyresource-pod.slice` is a hard code value from kubelet?